internal/input/accounts.go

```diff
@@ -0,0 +1,79 @@
+package input
+
+import (
+	"context"
+
+	"github.com/stellar/stellar-etl/internal/utils"
+
+	"github.com/stellar/go/exp/ingest/adapters"
+	ingestio "github.com/stellar/go/exp/ingest/io"
+	"github.com/stellar/go/historyarchive"
+	"github.com/stellar/go/xdr"
+)
+
+// GetAccounts returns a slice of ledger close metas for the ledgers in the provided range (inclusive on both ends)
+func GetAccounts(start, end uint32, limit int64) ([]xdr.LedgerEntry, error) {
+	archiveStellarURL := "http://history.stellar.org/prd/core-live/core_live_001"
```

Comment:
Pull this out of the function as a package-wide variable - it'll be helpful for other commands.